pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -632,13 +633,28 @@ func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, devi
 				return nil, nil
 			}
 
-			diskFormatter := &mount.SafeFormatAndMount{
-				Interface: mounter,
-				Exec:      og.volumePluginMgr.Host.GetExec(expandableVolumePlugin.GetPluginName()),
-			}
+			var resizeStatus bool
+			var resizeErr error
+
+			if fsResizablePlugin, callExpandFs := expandableVolumePlugin.(volume.FSResizableVolumePlugin); callExpandFs {
+				// special case for flex volumes
+				if _, resizeErr = fsResizablePlugin.ExpandVolumeDevice(volumeToMount.VolumeSpec, pvSpecCap, pvcStatusCap); resizeErr == nil {
+					resizeErr = fsResizablePlugin.ExpandFS(volumeToMount.VolumeSpec, devicePath, deviceMountPath)
```

Comment:
`ExpandFS` call should be extended to all volume plugins that require file system resize rather than special casing this for Flex.

Reply:
@aniket-s-kulkarni Yes this would be required for all plugins going forward.

Reply:
OK so we go ahead and add this to all existing plugins.

Reply:
yeah. Also all the existing plugins will simply call out to existing util resize function.